### ISSUE TITLE
default output for ->find_no_element_ok

### ISSUE THREAD

**Issue by davewood** [teodesian/Selenium-Remote-Driver#335]
$driver->find_no_element_ok( "testcontact-hzsbvpze", "link_text" );

**Comment by teodesian** [comment]
Whoever thought we should provide the assertion text needs to have their heads examined.  Anything we can do automatically is guaranteed to be meaningless -- whee, I couldn't find an element with some selector.  So?  Does that mean I can figure out any better what the actual problem is?  No.

Nevertheless, for those desiring useless output, I guess this would be better than what it does already, which is not much.  I'll look at adjusting the assert text for this next release.

**Comment by davewood** [comment]
thanks for your reply, this seems a bit missformatted aswell. note the misbalanced quotes

```
my $xpath = q{//contact-relationships//a[text()="testcontact-fiykcqgg"]};
$driver->find_element_ok($xpath, "xpath");
```

> ok 5 - find_element'//contact-relationships//a[text()="testcontact-fiykcqgg"]' xpath'

**Comment by teodesian** [comment]
yep.  I'm guessing the DWIM for the above example is to make sure the selector and type printed are copy/pastable without modification.  I'll look at ensuring that is the case.

**Comment by teodesian** [comment]
Notes: T::S::R::D::Role::DoesTesting is basically just shooting out the args to _check_ok(), which are basically the args to whatever finder method it's using.  It looks so dumb due to it just being an interpolated @args array.

I'll have to make a new handler sub to try and "do the right thing" in the context.  This would have been less work if the module had been designed as straightforward wrappers rather than this too-clever by half use of Sub::Install.  But, it's too late to change that now.

**Comment by teodesian** [comment]
Feel free to ignore my babbling, I tend to leave notes in cases for later use -- It's usually the odd couple of hours every few days I can look at this stuff so I gotta record my findings when I can.  It also usually helps to sleep on problems like this.  In any case, I think I can make this code even simpler and fix a bug (it looks as though whether or not we find/don't find the element we probably return true (oh no!))
